x-pack/plugins/security/public/nav_control/nav_control_component.tsx

```diff
@@ -105,6 +105,9 @@ export const SecurityNavControl: FunctionComponent<SecurityNavControlProps> = ({
       ),
       icon: <EuiIcon type={hasCustomProfileLinks ? 'controlsHorizontal' : 'user'} size="m" />,
       href: editProfileUrl,
+      onClick: () => {
+        setIsPopoverOpen(false);
+      },
```

Comment:
This is the only material change in this file. The rest is a rename from `isOpen` -> `isPopoverOpen`.